tests/end-to-end/backup/pkg/common/common.go

```diff
@@ -99,53 +96,60 @@ func TestBackupAndRestoreCluster(t *testing.T) {
 	for idx, backupTest := range backupTests {
 
 		name := string("")
-		if t := reflect.TypeOf(backupTest); t.Kind() == reflect.Ptr {
+		if t := reflect.TypeOf(backupTest.backupTest); t.Kind() == reflect.Ptr {
 			name = t.Elem().Name()
 		} else {
 			name = t.Name()
 		}
 
 		e2eTests[idx] = e2eTest{
+			backupTest: backupTest.backupTest,
+			enabled:    backupTest.enabled,
 			name:       name,
-			backupTest: backupTest,
 			namespace:  fmt.Sprintf("%s-backup-test", strings.ToLower(name)),
 		}
 	}
 
 	myBackupClient, err := client.NewBackupClient()
 	fatalOnError(t, err, "while creating custom client for Backup")
 
-	switch action {
-	case testBeforeBackupActionName:
-		Convey("Create resources\n", t, func() {
-			for _, e2eTest := range e2eTests {
+	switch mode {
+	case TestBeforeBackup:
+		for _, e2eTest := range e2eTests {
+			if !e2eTest.enabled {
+				logrus.Infof("Skipping %v", e2eTest.namespace)
+				continue
+			}
+			convey.Convey(fmt.Sprintf("Create resources for %v", e2eTest.namespace), t, func() {
+				logrus.Infof("Creating Namespace: %s", e2eTest.namespace)
 				t.Logf("Creating Namespace: %s", e2eTest.namespace)
 				err := myBackupClient.CreateNamespace(e2eTest.namespace)
-				So(err, ShouldBeNil)
+				convey.So(err, convey.ShouldBeNil)
 				t.Logf("[CreateResources: %s] Starting execution", e2eTest.name)
 				e2eTest.backupTest.CreateResources(e2eTest.namespace)
-				t.Logf("[CreateResources: %s] End with success", e2eTest.name)
-			}
-			for _, e2eTest := range e2eTests {
-				t.Logf("[TestResources: %s] Starting execution", e2eTest.name)
+				logrus.Infof("Testing resources in namespace: %s", e2eTest.namespace)
```

Comment:
Printing the test name would be better than printing the namespace.

Reply:
oh, i missed that